app/views/users/_table.html.erb

```diff
@@ -14,6 +15,7 @@
         <td><%= user.first_name %></td>
         <td><%= user.last_name %></td>
         <td><%= user.spire_id %></td>
+        <td><%= user.department.name %></td>
```

Comment:
Good, but we have some existing code that allows for `nil` departments here: https://github.com/umts/probable-engine/blob/master/app/models/user.rb#L15

I am OK with requiring a department at a model and database level, or wrapping the `.name` call in a `.try` method for the time being.